src/main/java/Duke.java

```diff
@@ -1,10 +1,154 @@
+import java.util.Scanner;
 public class Duke {
-    public static void main(String[] args) {
+
+    public static Task[] tasks = new Task[100];
+    public static int numTasks = 0;
+
+    public static void listTasks(){
+        System.out.println("Beep beep, listing out the tasks....Loading.....");
+        for(int i = 0; i < numTasks; i++){
+            System.out.println((i + 1) + "." + tasks[i].toString());
+        }
+    }
+
+    public static void addTask(String newTask){
+        tasks[numTasks] = new Task(newTask);
+        System.out.println("added: " + newTask);
+        numTasks += 1;
+        System.out.println("Beep boop, now you have " + numTasks + " tasks");
+
+    }
+
+    public static void addTodo(String newToDo){
+        tasks[numTasks] = new Todo(newToDo);
+        System.out.println("HELLO BEEP, added a new ToDo: ");
+        System.out.println("\t" + tasks[numTasks].toString());
+        numTasks += 1;
+        System.out.println("Beep boop, now you have " + numTasks + " tasks");
+    }
+
+    public  static void addDeadline(String param){
```

Comment:
perhaps it is accidental, but there are actually 2 spaces between 'public' and 'static' in this method and the next method